rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -1085,6 +1085,7 @@ std::vector<rocalTensorList *> MasterGraph::create_coco_meta_data_reader(const c
     _meta_data_graph = create_meta_data_graph(config);
     _meta_data_reader = create_meta_data_reader(config, _augmented_meta_data);
     _meta_data_reader->read_all(source_path);
+    auto max_img_size = _meta_data_reader->get_max_size();
```

Comment:
Move this inside the if condition for pixel wise mask, Reduce the scope of this variable